### ISSUE TITLE
[BUG] Upgrade to 1.2.2 connects locally but wallet is in loop

### ISSUE THREAD

**Issue by Dartellum** [Chia-Network/chia-blockchain#7503]
**Describe the bug**

The wallet fetch routine is throwing an error, over and over: 

2021-07-14T16:27:53.470 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8449}
2021-07-14T16:27:53.471 wallet chia.wallet.wallet_node    : ERROR    Error while trying to fetch from peer:Peer returned no response Traceba>
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 527, in batch_sync_to_peak
    added, advanced_peak = await self.fetch_blocks_and_validate(
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 679, in fetch_blocks_and_validate
    raise ValueError("Peer returned no response")
ValueError: Peer returned no response

2021-07-14T16:27:53.471 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 0-32. Traceback (most>
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 580, in sync_job
    await self._sync()
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 660, in _sync
    await self.batch_sync_to_peak(fork_height, peak)
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 537, in batch_sync_to_peak
    raise RuntimeError(f"Was not able to add blocks {start_height}-{end_height}")
RuntimeError: Was not able to add blocks 0-32

2021-07-14T16:27:53.471 wallet chia.wallet.wallet_node    : INFO     Loop end in sync job
2021-07-14T16:27:53.471 wallet chia.wallet.wallet_node    : INFO     Loop start in sync job
2021-07-14T16:27:53.516 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}   
2021-07-14T16:27:53.532 wallet wallet_server              : INFO     Connected with full_node {'host': '127.0.0.1', 'port': 8444}
2021-07-14T16:27:56.382 wallet chia.full_node.weight_proof: INFO     validating 793 sub epochs
2021-07-14T16:27:56.383 wallet chia.wallet.wallet_node    : INFO     Validated, fork point is 0
2021-07-14T16:27:56.383 wallet chia.wallet.wallet_node    : INFO     self.sync_event.set()
2021-07-14T16:27:56.401 wallet chia.wallet.wallet_node    : INFO     Have collected 1 potential peaks
2021-07-14T16:27:56.402 wallet chia.wallet.wallet_node    : INFO     Requesting blocks 0-32

**To Reproduce**
Upgrade to 1.2.2

1. I started from fresh install by removing the `chia-blockchain` folder after stopping the services. I tried the upgrade path but 1.2.2 was not coming down as latest when I did `git checkout latest`
2. Start the farm: `. ./activate` then `chia start farmer`
4. Open debug.log to see errors

**Expected behavior**

Expect the program to pickup from where it was and start syncing the new data.

**Screenshots**

**Desktop**

- OS:  Linux
- OS Version/Flavor:  Ubuntu 20.04
- CPU:  Intel i7 3770K

**Additional context**

**Comment by Dartellum** [comment]
It also did not like my wallet db backup I made before the upgrade after I shutdown all chia services:
```
2021-07-14T17:13:52.606 wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "/home/chia/chia-blockchain/chia/rpc/rpc_server.py", line 81, in inner
    res_object = await f(request_data)
  File "/home/chia/chia-blockchain/chia/rpc/wallet_rpc_api.py", line 167, in log_in
    started = await self.service._start(fingerprint=fingerprint, backup_file=file_path)
  File "/home/chia/chia-blockchain/chia/wallet/wallet_node.py", line 174, in _start
    await self.wallet_state_manager.import_backup_info(backup_file)
  File "/home/chia/chia-blockchain/chia/wallet/wallet_state_manager.py", line 1089, in import_backup_info   
    json_dict = open_backup_file(file_path, self.private_key)
  File "/home/chia/chia-blockchain/chia/wallet/util/backup_utils.py", line 18, in open_backup_file  
    backup_file_text = file_path.read_text()
  File "/usr/lib/python3.8/pathlib.py", line 1237, in read_text
    return f.read()
  File "/usr/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd1 in position 30: invalid continuation byte
```

**Comment by Dartellum** [comment]
Still doing this. Any updates I can apply to fix this? I am on 1.2.2.

**Comment by Dartellum** [comment]
I removed the wallet dbs and it started syncing from 0. Resolving this, for now. However, still would like to see this fixed. What is the point in having a db backup if it does not work?
